instance/models/openedx_appserver.py

```diff
@@ -144,6 +154,8 @@ def set_field_defaults(self):
         """
         Set default values.
         """
+        if not self.github_admin_organizations and settings.DEFAULT_ADMIN_ORGANIZATION:
```

Comment:
Can we change this to `if not (self.github_admin_organizations or self.github_admin_users) and settings.DEFAULT_ADMIN_ORGANIZATION`? If any users have access, I think we shouldn't set a default. @antoviaque or @bradenmacdonald may have an opinion on that though.